<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="AsiF" name="AppTestRunner.ExperiBankTestRunner" tests="4" failures="0" timestamp="2023-05-29T22:53:25 BDT" time="34.771" errors="4">
  <testcase name="runScenario" time="0.006" classname="AppTestRunner.ExperiBankTestRunner">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at POM.AppElements.expenseReport(AppElements.java:154)
at AppStepDefinition.EB4ExpenseReportStepDefs.userTapsOnTheExpenseReportButton(EB4ExpenseReportStepDefs.java:17)
at ✽.user taps on the expense report button(file:///D:/Apps/Programming%20Language/Selenium/E2EAutomation/PhoneAppAutomationCucumber/src/test/resources/Features/EB4ExpenseReport.feature:4)
]]>
    </error>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="33.226" classname="AppTestRunner.ExperiBankTestRunner">
    <error type="org.openqa.selenium.SessionNotCreatedException" message="Could not start a new session. Response code 500. Message: An unknown server-side error occurred while processing the command. Original error: The instrumentation process cannot be initialized. Make sure the application under test does not crash and investigate the logcat output. 
Host info: host: &#039;ASIF&#039;, ip: &#039;192.168.0.103&#039;
Build info: version: &#039;4.7.0&#039;, revision: &#039;0a5b49d16f&#039;
System info: os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;1.8.0_202&#039;
Driver info: io.appium.java_client.android.AndroidDriver
Command: [null, newSession {capabilities=[{appium:appActivity=.LoginActivity, appium:appPackage=com.experitest.ExperiBank, appium:deviceName=adb:SM-J700H, platformName=ANDROID, appium:platformVersion=6.0, appium:udid=3300a9bf997622c3}], desiredCapabilities=Capabilities {appActivity: .LoginActivity, appPackage: com.experitest.ExperiBank, deviceName: adb:SM-J700H, platformName: ANDROID, platformVersion: 6.0, udid: 3300a9bf997622c3}}]
Capabilities {}">
      <![CDATA[org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Response code 500. Message: An unknown server-side error occurred while processing the command. Original error: The instrumentation process cannot be initialized. Make sure the application under test does not crash and investigate the logcat output. 
Host info: host: 'ASIF', ip: '192.168.0.103'
Build info: version: '4.7.0', revision: '0a5b49d16f'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_202'
Driver info: io.appium.java_client.android.AndroidDriver
Command: [null, newSession {capabilities=[{appium:appActivity=.LoginActivity, appium:appPackage=com.experitest.ExperiBank, appium:deviceName=adb:SM-J700H, platformName=ANDROID, appium:platformVersion=6.0, appium:udid=3300a9bf997622c3}], desiredCapabilities=Capabilities {appActivity: .LoginActivity, appPackage: com.experitest.ExperiBank, deviceName: adb:SM-J700H, platformName: ANDROID, platformVersion: 6.0, udid: 3300a9bf997622c3}}]
Capabilities {}
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:148)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at io.appium.java_client.remote.AppiumProtocolHandshake.createSession(AppiumProtocolHandshake.java:133)
at io.appium.java_client.remote.AppiumProtocolHandshake.createSession(AppiumProtocolHandshake.java:102)
at io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:194)
at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:262)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:541)
at io.appium.java_client.AppiumDriver.startSession(AppiumDriver.java:229)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:157)
at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:80)
at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:92)
at io.appium.java_client.android.AndroidDriver.<init>(AndroidDriver.java:117)
at Core.Helper.apkLaunch(Helper.java:26)
at AppStepDefinition.EB1LoginStepDefs.appApkIsThere(EB1LoginStepDefs.java:18)
at ✽.app apk is there(file:///D:/Apps/Programming%20Language/Selenium/E2EAutomation/PhoneAppAutomationCucumber/src/test/resources/Features/EB1Login.feature:5)
]]>
    </error>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="0.010" classname="AppTestRunner.ExperiBankTestRunner">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at POM.AppElements.makePaymentTab(AppElements.java:50)
at AppStepDefinition.EB2MakePaymentStepDefs.userClicksOnThrMakePaymentTab(EB2MakePaymentStepDefs.java:18)
at ✽.user clicks on thr make payment tab(file:///D:/Apps/Programming%20Language/Selenium/E2EAutomation/PhoneAppAutomationCucumber/src/test/resources/Features/EB2MakePayment.feature:5)
]]>
    </error>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="1.529" classname="AppTestRunner.ExperiBankTestRunner">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at POM.AppElements.mortgageRequest(AppElements.java:91)
at AppStepDefinition.EB3MortgageRequestStepDefs.userClicksOnTheMortgageRequestTab(EB3MortgageRequestStepDefs.java:18)
at ✽.user clicks on the mortgage request tab(file:///D:/Apps/Programming%20Language/Selenium/E2EAutomation/PhoneAppAutomationCucumber/src/test/resources/Features/EB3MortgageRequest.feature:4)
]]>
    </error>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- AppTestRunner.ExperiBankTestRunner -->
